src/shared/mount-util.c

```diff
@@ -1010,11 +1011,9 @@ static int make_userns(uid_t uid_shift, uid_t uid_range) {
         r = safe_fork("(sd-mkuserns)", FORK_CLOSE_ALL_FDS|FORK_DEATHSIG|FORK_NEW_USERNS, &pid);
         if (r < 0)
                 return r;
-        if (r == 0) {
+        if (r == 0)
                 /* Child. We do nothing here, just freeze until somebody kills us. */
                 freeze();
-                _exit(EXIT_FAILURE);
```

Comment:
wouldn't it be better to leave this? Just in case - doesn't really hurt, no?

Reply:
We don't do that in other places, e.g. freeze_or_exit_or_reboot() in main. I think if we trust the function in PID1, then we should also here.

Reply:
fair enough